src/config/example.toml

```diff
@@ -171,9 +171,13 @@ reclaimers = 4
 recover_concurrency = 8
 insert_rate_limit_mb = 0
 indexer_shards = 64
-compression = "none"
+compression = "None"
```

Comment:
Is lower cased string still supported? (I don't care. Just ask)

Reply:
No, but I think nobody is using this field for now. And none is the default configuration. It's okay to switch to serde defauts now.

Reply:
case-insensive config is slightly more user-friendly to me (both OK). But it seems other options (like `algorithm = "Lru"`, `recover_mode = "None"`) are also case-sensive, so might be better to keep them consistent.

Reply:
+1 Let me try.